tests/integration/test_main_window.py

```
@@ -0,0 +1,309 @@
+    def test_next_image(self):
+        """Test navigating to next image."""
+        initial_path = self.win.file_path
+        self.win.open_next_image()
+        self.assertNotEqual(self.win.file_path, initial_path)
+        self.assertEqual(self.win.file_path, self.image_paths[1])
+
+    def test_prev_image(self):
+        """Test navigating to previous image."""
+        # Start from second image
+        self.win.load_file(self.image_paths[1])
+        initial_path = self.win.file_path
```

Ruff (0.14.14)

[error] Local variable `initial_path` is assigned to but never used

Remove assignment to unused variable `initial_path`

(F841)